backend/tests/owasp/models/post_test.py

```
@@ -0,0 +1,80 @@
+        ("title", "expected_str"),
+        [
+            ("Sample Post", "Sample Post"),
+            ("", ""),
+        ],
+    )
+    def test_post_str(self, title, expected_str):
+        """Test the __str__ method of the Post model."""
+        post = Post(
+            title=title,
+            url="https://example.com",
+            published_at=datetime(2025, 1, 1)
```

Ruff (0.8.2)

`datetime.datetime()` called without a `tzinfo` argument

(DTZ001)